api/db-cdr/generate-cdr/make-bq-public-data.sh

```diff
@@ -109,13 +109,24 @@ set count_value =
     case when count_value < ${BIN_SIZE}
         then ${BIN_SIZE}
     else
-        cast(ROUND(count_value / ${BIN_SIZE}) * ${BIN_SIZE} as int64)
+        cast(CEIL(count_value / ${BIN_SIZE}) * ${BIN_SIZE} as int64)
```

Comment:
Can you update the comment that says how we use ceil instead of rounding now ? Its above this somewhere.

Reply:
Added it just beside round comment.